xxxx-release-and-versioning.md

```diff
@@ -27,45 +37,51 @@ Only a the latest `MAJOR.MINOR.PATCH` version of the last 2 `MAJOR.MINOR` versio
-
-Once a stable release will happen, a stable release shall have its own branch in a form of `stable-MAJOR.MINOR` and tag with `vMAJOR.MINOR.PATCH` as appropriate.
+The branching model will utilize a `master` branch, a stable version branch and a feature branch. To demostrate:
 
 ```
-          a1    b1   a2   b2   a3   c1   c2   c3   a4   b3
-develop ---o----o----o----o----o----o----o----o----o----o---------->
-           |    |    |    |    |    |    |    |
-           +--= | =--+    |    |    +----+----+
-                |    |    |    |              |
-                \--= | =--+--= | =--+         |
-                     |         |    |         |
-                     v         v    v         v
- stable -------------o---------o----o---------o-------------------->
-                  {a1,a2} \   {a3} {b1,b2}   {c1,c2,c3} \
-                           \    \                        \
-                            \    +------------            \
-                             \                \            \
- stable-1.0                   t----------o-----t------------------->
-                         v1.0.0 {a1,a2}     v1.0.1 {a3}      \
-                                                              \
- stable-1.1                                                    t--->
-                                                             v1.1.0
+feature2    -o---------o----------o
+                                   \
+                                    \
+feature1    -o                       \
+              \                       \
+               v                       v
+master      ----o-----------------------o-------------------------------->
+                             \                           \
+                              v                           \
+stable-1.0                     o-----------o----o----------^------------->
+                            v1.0.0           v1.0.1         \
+                                                             v
+stable-1.1                                                    o---------->
+                                                            v1.1.0
 ```
 
-In this chart, changes are selectively backported from `develop` to `stable` in a per-feature basis (`{a1,a2}`, `{b1,b2}`, ...) This process can be done by cherry-picking changes and fixing conflicts as necessary.
+#### Feature Branches
+
+Feature branches, such as `feature2` and `feature1`, are development branches that implements new feature or change existing functionality. These branches may be long-running and should be merged to `master` only when the feature is considered stable enough for public consumption. Feature branches should regularly sync up with master to avoid too much code diversion. Feature branches may have its own public testing environment for each branch with its data copied from staging on each deploy.
+
+#### `master` branch
```

Comment:
I feel having `master` has development branch is a bit weird, but it's probably my personal experience talking. Using `develop` would probably make it clearer for people that they shouldn't use that branch directly 🤔

Reply:
At the same time, having a repo without `master` would probably be weird 🤷‍♂️